src/i18n/de/ui.ts

```diff
@@ -58,16 +71,54 @@ export default UIDictionary({
 	'aside.tip': 'Tipp',
 	'aside.caution': 'Achtung',
 	'aside.danger': 'Gefahr',
+	// `<LanguageSelect>` vocabulary
+	'languageSelect.label': 'Sprache auswählen',
+  	// Integrations vocabulary
+	'integrations.changelog': 'Changelog',
+	'integrations.footerTitle': 'Weitere Integrationen',
+	'integrations.renderers': 'UI-Frameworks',
+	'integrations.adapters': 'SSR-Adapter',
+	'integrations.others': 'Sonstiges',
+  	// Checklist component
+	'checklist.or': 'oder',
+  	// Multiple Choice component
+	'multipleChoice.defaultCorrect': 'Korrekt!',
+  	'multipleChoice.defaultIncorrect': "Versuch's nochmal!",
+  	'multipleChoice.submitLabel': 'Absenden',
+  	// Tutorial Progress
+	'progress.todo': 'To-do',
+  	'progress.done': 'Erledigt',
+  	// Tutorial Navigation
+  	'tutorial.trackerLabel': 'Tutorial Tracker',
+	'tutorial.unit': 'Teil',
+  	// Tutorial
+	'tutorial.getReady': 'Mach dich bereit,…',
```

Comment:
```suggestion
	'tutorial.getReady': 'Mach dich bereit, …',
```

Nach Kommas folgt ein Leerzeichen. Es sieht in beiden Versionen irgendwie seltsam aus, aber so ist es richtiger. :)